src/python/pants/core/util_rules/system_binaries.py

```diff
@@ -687,6 +695,22 @@ async def find_mkdir() -> MkdirBinary:
     return MkdirBinary(first_path.path, first_path.fingerprint)
 
 
+@rule(desc="Finding the `cp` binary", level=LogLevel.DEBUG)
```

Comment:
I think that both of these are `bash` builtins, and so using them directly as unqualified in the shell is fine. Did you see otherwise?

Reply:
Yeah, I hit the error(s). Also not listed here: https://github.com/pantsbuild/pants/blob/main/src/python/pants/backend/shell/builtin.py